reference/gdscript.rst

```diff
@@ -114,7 +115,7 @@ here's a simple example of how it looks.
         print(lv.a)
 
 If you have previous experience with statically typed languages such as
-C, C++, or C# but never used a dynamically typed one, it is advised you
+C, C++, or C# but never used a dynamically typed one before, it is advised you
 read this tutorial: :ref:`doc_gdscript_more_efficiently`.
```

Comment:
Not part of this file, but "*GDScript more efficiently*" is IMO a misleading name for the linked tutorial. Its name suggests to me that I should go there to learn how to optimize my GDScript code (that would be an interesting page in its own right). But as it is, that page should probably be renamed to "*Introduction to dynamic programming languages*".